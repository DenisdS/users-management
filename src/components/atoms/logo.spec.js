import React from 'react'
import { shallow } from '../../enzyme'

import imgLogo from '../../assets/logo.png'
import Logo from './logo'

let wrapper

beforeEach(() => {
  wrapper = shallow( <Logo /> );
});

describe('<Logo /> rendering', () => {

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("Logo renders an image", () => {
    const logo = shallow(<Logo />);

    expect(logo.find("img").prop("src")).toEqual(imgLogo);
  });

})




