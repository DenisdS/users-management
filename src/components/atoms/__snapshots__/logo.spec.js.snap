// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> rendering renders correctly 1`] = `ShallowWrapper {}`;
